src/test/java/org/folio/linked/data/validation/dto/PrimaryTitleDtoValidatorTest.java

```diff
@@ -0,0 +1,95 @@
+package org.folio.linked.data.validation.dto;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.folio.linked.data.domain.dto.ParallelTitle;
+import org.folio.linked.data.domain.dto.ParallelTitleField;
+import org.folio.linked.data.domain.dto.PrimaryTitle;
+import org.folio.linked.data.domain.dto.PrimaryTitleField;
+import org.folio.linked.data.domain.dto.TitleField;
+import org.folio.spring.testing.type.UnitTest;
+import org.junit.jupiter.api.Test;
+
+@UnitTest
+class PrimaryTitleDtoValidatorTest {
+
+  private final PrimaryTitleDtoValidator validator = new PrimaryTitleDtoValidator();
+
+  @Test
+  void shouldReturnFalse_ifGivenTitleFieldListIsNull() {
+    // given
+    List<TitleField> titleFields = null;
+
+    // when
+    boolean result = validator.isValid(titleFields, null);
+
+    // then
+    assertThat(result).isFalse();
```

Comment:
it's violate of validation API contract, which I describe in class
this test with null value must be like `assertThat(result).isTrue();`

Reply:
fixed